deployment-scanner/src/main/java/org/jboss/as/server/deployment/scanner/DeploymentScannerService.java

```diff
@@ -178,6 +178,7 @@ public DeploymentOperations create() {
      */
     @Override
     public synchronized void stop(StopContext context) {
+        context.asynchronous();
```

Comment:
I'm not also but the scanner.stop is happening in another thread I understand correctly.